Title: avformat/mxfenc: fix index byte count in partition header
Author: Baptiste Coudurier <baptiste.coudurier@gmail.com>
Date:   Thu Jul 18 10:35:00 2019 -0700
Forwarded: https://trac.ffmpeg.org/ticket/9036

--- a/libavformat/mxfenc.c
+++ b/libavformat/mxfenc.c
@@ -1857,8 +1857,7 @@
         index_byte_count = 80;
 
     if (index_byte_count) {
-        // add encoded ber length
-        index_byte_count += 16 + klv_ber_length(index_byte_count);
+        index_byte_count += 16 + 4; // add encoded ber4 length
         index_byte_count += klv_fill_size(index_byte_count);
     }
 
